packages/shared/src/config/modules.ts

```diff
@@ -76,17 +80,18 @@ export const puzzlesBanner: ModuleInfo = getDefaultModuleInfo(
 export const header: ModuleInfo = getDefaultModuleInfo('header', 'header/Header');
 
 export const moduleInfos: ModuleInfo[] = [
-    epic,
-    liveblogEpic,
-    contributionsBanner,
-    contributionsBannerWithSignIn,
-    investigationsMomentBanner,
-    environmentMomentBanner,
-    usEoyMomentBanner,
-    usEoyMomentGivingTuesdayBanner,
-    globalNewYearBanner,
-    digiSubs,
-    guardianWeekly,
-    puzzlesBanner,
-    header,
+    // epic,
```

Comment:
Are all of these supposed to be commented out?

Reply:
nope!

Reply:
right pushed that up!